src/orion/core/cli/hunt.py

```diff
@@ -59,6 +59,10 @@ def main(args):
     args['leafs'] = []
     # TODO: simplify when parameter parsing is refactored
     experiment = experiment_builder.build_from_args(args)
+
+    if len(experiment.space) == 0:
```

Comment:
This problem would affect `init_only` as well. We could include this test in experiment_builder.build instead.

Reply:
Ok! Wasn't really sure where to test. I will change this. This will allow me to also display the help instead of an exception :)